<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>video_widget</class>
    <widget class="QWidget" name="video_widget">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>900</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>video</string>
        </property>
        <widget class="QLabel" name="video_label">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>900</width>
                    <height>600</height>
                </rect>
            </property>
            <property name="text">
                <string/>
            </property>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
